### ISSUE TITLE
Do not include platform name in executable filename

### ISSUE THREAD

**Issue by PanderMusubi** [gotify/server#537]
**Is your feature request related to a problem? Please describe.**
When using different hardware platforms, after unzipping a release, the filename of the executable needs to be renamed in order to match startup scripts.

**Describe the solution you'd like**
Release the server executables without a platform name in the filename.

**Describe alternatives you've considered**
n/a

**Additional context**
Actually, there is an even nicer way to create release files which is used by many other projects which save work and mistakes during installing and upgrading. Please consider releasing in this way:

    gotify-2.2.3-linux-arm7.zip
    - gotify-2.2.3 (directory)
      - gotify (executable for arm7)
      - LICENSE
      - ...
    gotify-2.2.3-linux-amd64.zip
    - gotify-2.2.3 (directory)
      - gotify (executable for amd64)
      - LICENSE
      - ...

**Comment by jmattheis** [comment]
I agree, but let's keep it as is, as it will break install scripts that already depend on this structure.

**Comment by PanderMusubi** [comment]
Okay, keep this issue to change it in the future.

**Comment by jmattheis** [comment]
Well, I don't really see that this has to be changed, the startup script should be able to rename the binary to something that you need. So I'll close this ticket.
